activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+*   `AR::Relation#reverse_order` throws `AR::IrreversibleOrderError` when the order can not be reversed
```

Comment:
If it's so obvious what library these constants belong to, why even put AR there in the first place? :smile:

There's precedent in the file to either spell it out or leave it out. Go with one of those.